packages/general/src/util/Observable.ts

```diff
@@ -184,11 +185,7 @@ export class BasicObservable<T extends any[] = any[], R = void> implements Obser
                 try {
                     result = observer(...payload);
                 } catch (e) {
-                    if (e instanceof Error) {
-                        this.#errorHandler(e, observer);
-                    } else {
-                        this.#errorHandler(new Error(`${e}`), observer);
-                    }
+                    this.#errorHandler(asError(e, "Error processing Observables"), observer);
```

Comment:
I don't think there's should be a prefix here, rather just assume that if someone threw a non-error the value will be meaningful to them